queue/src/fiskaltrust.Middleware.Localization.QueueDE/QueueDEBootstrapper.cs

```diff
@@ -50,9 +57,20 @@ public void ConfigureServices(IServiceCollection services)
                     tarFileCleanupService.CleanupAllTarFilesAsync().Wait();
 
                     return tarFileCleanupService;
-                })
-                .AddSingleton<IRequestCommandFactory, RequestCommandFactory>()
-                .ConfigureReceiptCommands();
+                });
+        }
+
+        public Task<Func<IServiceProvider, Task>> ConfigureServicesAsync(IServiceCollection services)
+        {
+            var _ = ConfigureServicesInner(services)
+                .AddSingleton<IDESSCDProvider, DESSCDProvider>()
+                .AddSingleton<ITarFileCleanupService, TarFileCleanupService>();
+
+            return Task.FromResult<Func<IServiceProvider, Task>>(async (IServiceProvider services) =>
+            {
+                await services.GetRequiredService<IDESSCDProvider>().RegisterCurrentScuAsync();
```

Comment:
So I guess we could discuss this in another issue, but IMO we should not register the scu in our configure services method.. also probably not during resolution of the instance, but more or less during the first creation. By doing that we can probably get rid of some tasking things